### PR TITLE
add adelais rev4

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

Add Adelais rev.4.
Thanks.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by mechlovin** [comment]
bump

File: keyboards/mechlovin/adelais/standard_led/rev4/config.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#pragma once
```
Comment (drashna) [inline keyboards/mechlovin/adelais/standard_led/rev4/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/adelais/standard_led/rev4/config.h (hunk 1 above)
Comment (mechlovin) [reply keyboards/mechlovin/adelais/standard_led/rev4/config.h:1]: sure, thanks.

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Also:
```
☒ Missing keyboards/mechlovin/adelais/standard_led/readme.md
☒ Lint check failed!
linting mechlovin/adelais/standard_led/rev4
☒ Missing keyboards/mechlovin/adelais/standard_led/rev4/readme.md
☒ Lint check failed!
```

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**